serve_configs/amazon--LightGPT.yaml

```diff
@@ -1,6 +1,6 @@
 applications:
 - name: amazon--LightGPT
```

Comment:
I suggest removing the application names as well

Reply:
Sounds good, I changed all the names to `ray-llm`.